semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -605,6 +604,15 @@ trait TextDocumentOps { self: SemanticdbOps =>
             }
 
             gtree match {
+              case v: g.ValDef
```

Comment:
Can we actually match on `g.Apply` ? Since that is a tree that we are actually interested in.

Reply:
Actually - yes, we can, changed it.